lib/mobility-core/src/Kernel/External/Maps/Interface/Google.hs

```diff
@@ -284,3 +286,24 @@ getPlaceName cfg GetPlaceNameReq {..} = do
     (mbByPlaceId, mbByLatLong) = case getBy of
       ByPlaceId id -> (Just id, Nothing)
       ByLatLong latLong -> (Nothing, Just latLong)
+
+getArea :: Text -> Text
+getArea fullArea = do
+  T.intercalate "," $ List.takeEnd 2 $ List.dropEnd 1 (T.split (== ',') fullArea)
+
+getNearBySearch ::
+  ( EncFlow m r,
+    CoreMetrics m
+  ) =>
+  GoogleCfg ->
+  GetNearBySearchReq ->
+  m Text
```

Comment:
can you send whole API response instead of just the first vicinity field, don't filter response in library we can filter in beckn repo based on use case.